### DISCUSSION TITLE
How to re-authenticate when using onedrive as a service

### DISCUSSION THREAD

**Discussion by meskaya** [abraunegg/onedrive#1858] [Q&A] [answered]
Hi all,
I am not sure how I should re-authenticate when my token has expired as using --logout as stated [here](https://github.com/abraunegg/onedrive/blob/0158f356dbbf91c3c2222ce37b1425d63c20c596/docs/USAGE.md#handling-a-onedrive-account-password-change) doesn't work anymore.
Re-auth has been disabled with this pull request https://github.com/abraunegg/onedrive/pull/1705 
Thanks :)

**Comment by abraunegg** [comment, marked as answer]
@meskaya 
> I am not sure how I should re-authenticate

Simply run the application on its own and it will request to re-auth.

**Reply by meskaya** [reply]
@abraunegg Thanks
I just run onedrive without additional command switch and was able to re-auth

**Comment by abraunegg** [comment]
@meskaya 
PR #1867 introduced `--reauth` to simplify this process to clean up the old authentication and prompt for reauth

**Reply by meskaya** [reply]
@abraunegg great I just tried it and it worked perfectly.
Thanks !!!
